### DISCUSSION TITLE
Emails lack date

### DISCUSSION THREAD

**Discussion by lheckemann** [community/community#32906] [Actions]
Emails sent from GitHub Actions (e.g. "Run failed") lack a `Date` header, which is required by [Section 3.6 of RFC 5322](https://www.rfc-editor.org/rfc/rfc5322#section-3.6).

**Comment by Akash1134** [comment]
Hi @lheckemann, Thanks for reporting this behaviour. What is the current environment you're running the Actions in and what format is the date set to along with the time offset? Does the `Date` header works as expected with other workflows?

**Reply by lheckemann** [reply]
I'm not sure what "the environment" refers to? `runs-on` for the job is set to `ubuntu-latest`.

I've also just noticed that not all "Run failed" notification emails have this problem: between 2022-08-01 and today, I've received 9 such emails without a Date header, and 11 with.
